packages/create-cloudflare/src/deploy.ts

```diff
@@ -67,12 +68,11 @@ const isDeployable = async (ctx: C3Context) => {
 		return true;
 	}
 
-	const wranglerToml = readWranglerToml(ctx);
-	if (wranglerToml.match(/(?<!#\s*)bindings?\s*=.*/m)) {
```

Comment:
This approach was too simplistic because it also blocked Workers that have assets bindings, which can be deployed straight up.